<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FullScreenController</class>
 <widget class="QFrame" name="FullScreenController">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>75</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>75</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Sunken</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="SeekSlider" name="position">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="TimeLabel" name="timeLabel">
       <property name="text">
        <string>--:--</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="PlayButton" name="btn_playpause">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../QtVlc/vlc/modules/gui/qt4/vlc.qrc">
         <normaloff>:/toolbar/play_b</normaloff>:/toolbar/play_b</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btn_stop">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../QtVlc/vlc/modules/gui/qt4/vlc.qrc">
         <normaloff>:/toolbar/stop_b</normaloff>:/toolbar/stop_b</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="btn_togglewide">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../QtVlc/vlc/modules/gui/qt4/vlc.qrc">
         <normaloff>:/toolbar/space</normaloff>:/toolbar/space</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btn_defullscreen">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../QtVlc/vlc/modules/gui/qt4/vlc.qrc">
         <normaloff>:/toolbar/defullscreen</normaloff>:/toolbar/defullscreen</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="SoundWidget" name="volume" native="true"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TimeLabel</class>
   <extends>QLabel</extends>
   <header location="global">QtVlcWidgets/TimeLabel.h</header>
  </customwidget>
  <customwidget>
   <class>SeekSlider</class>
   <extends>QSlider</extends>
   <header location="global">QtVlcWidgets/SeekSlider.h</header>
  </customwidget>
  <customwidget>
   <class>SoundWidget</class>
   <extends>QWidget</extends>
   <header location="global">QtVlcWidgets/SoundWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PlayButton</class>
   <extends>QToolButton</extends>
   <header location="global">QtVlcWidgets/SpecialButtons.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../QtVlc/vlc/modules/gui/qt4/vlc.qrc"/>
 </resources>
 <connections/>
</ui>
